debian/control

```diff
@@ -97,6 +97,13 @@ Depends: nnstreamer, openvino, ${shlibs:Depends}, ${misc:Depends}
 Description: NNStreamer OpenVino support
  This package allows nnstreamer to support OpenVino.
 
+Package: nnstreamer-tvm
+Architecture: any
+Multi-Arch: same
+Depends: nnstreamer, tvm-runtime, ${shlibs:Depends}, ${misc:Depends}
```

Comment:
I think this dependency can be automatically resolved if `tvn-runtime-dev` is normal. Could you check?

Reply:
Yes indeed